lib/ckb/udt/base_wallet.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+module Ckb
+  module Udt
+    class BaseWallet
+      attr_reader :api
+      attr_reader :key
+      attr_reader :token_info
+
+      # privkey => hex
```

Comment:
What does this mean? Is `privkey` in bytes only?

Reply:
I forget to remove this line... it just means privkey is a hex string, not bin

Reply:
OK cool.